search-service/src/test/kotlin/com/egm/stellio/search/web/EntityAccessControlHandlerTests.kt

```diff
@@ -697,9 +711,12 @@ class EntityAccessControlHandlerTests {
             jsonLdEntity[AUTH_PROP_SAP] = JsonLdUtils.buildExpandedProperty(specificAccessPolicy)
         }
         rCanReadUser?.run {
```

Comment:
this `createJsonLdEntity` is having a lot of JSON-LD manipulation and this is not good. Maybe rewrite it to:
- create an `EntityAccessRight` object
- call its `serializeProperties` function
- returns this result wrapped in a `JsonLdEntity`